ERCS/erc-0000.md

```diff
@@ -0,0 +1,176 @@
+The ERC-xxx standard introduces a minimal interface for tokens to communicate cross-chain. It allows bridges with mint and burn rights to send and relay token transfers with a standardized API.
+
+## Motivation
+
+All rollups and multiple important sidechains (Gnosis and Polygon PoS, among others) implement canonical bridges that embed their security into some part of the network's core architecture. These bridges do not have mint/burn rights over original tokens, so they usually lock (unlock) liquidity on the native chain and then mint (burn) a non-equivalent representation on the other. Mint/burn is used because the native token is non-existent on that side, so they must create a new representation.
+
+However, each bridge implements a different interface for minting/burning on non-native chains. For example, L1 bridged tokens need to support the `OptimismMintableERC20` specification when bridged (minted) to the OP Stack, but the `IArbToken` interface when bridged to the Arbitrum Stack.
+
+This interface fragmentation is a massive issue in cross-chain communication among chains via third-party bridges or future canonical solutions. At this point, it is clear that every bridge would benefit from a standardized interface for minted/burnt tokens. 
+
+There have been different attempts in the past to standardize token-bridging interfaces. However, third-party providers are also developing cross-chain token frameworks, such as NTT by Wormhole and OFT by LayerZero. Each framework defines its features, like rate limits and fee switches, and implements its mint and burn versions. The resultant interfaces become highly specific, lacking naming conventions and structures.
+
+The proposed interface includes the most relevant and minimal set of actions used by most of these standards. These actions also do not require any governance or owner participation, in contrast, for instance, to set rate limits. xERC-20, OFT, NTT, and others can be built on top of this interface. 
+
+## Specification
+
+This ERC introduces the `IERCxxx` interface.
+
+### Methods
+
+**`crosschainMint`**
+
+Mints `_amount` of token to address `_account`. 
+
+This function works as the minting entry point for bridge contracts. Each implementation is responsible for its access control logic.
+
+```solidity
+crosschainMint(address _account, uint256 _amount)
+```
+
+**`crosschainBurn`**
+
+Burns `_amount` of token from address `_account`.
+
+This function works as the burning entry point for bridge contracts. Each implementation is responsible for its access control logic.
+
+```solidity
+crosschainBurn(address _account, uint256 _amount)
+```
+
+### Events
+
+**`CrosschainMint`**
+
+MUST trigger when `crosschainMint` is successfully called.
+
+```solidity
+event CrosschainMint(address indexed _to, uint256 _amount)
+```
+
+Note: implementations might consider additionally emitting `Transfer(address(0), _to, _amount)`to be compliant with [ERC-5679](./eip-5679.md).
+
+**`CrosschainBurnt`**
```

Comment:
```suggestion
**`CrosschainBurn`**
```

Can this be `CrosschainBurn` or change the `CrosschainMint` to `CrosschainMinted`? Whatever we land on the tense of the verb should be the same I think.

Reply:
Seems like this is meant to be `CrosschainBurn`, per [this comment](https://github.com/ethereum/ERCs/pull/692/files#r1823658960).